### PR TITLE
Cross addon

### PR DESCRIPTION
**Pull request by Sefiraat**
Allows items to be added to Netheo without warnings being thrown in the console as per https://github.com/Slimefun/Slimefun4/pull/3608 - Waited 9 days to allow servers to update first.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -125,7 +125,7 @@
         <dependency>
             <groupId>com.github.Slimefun</groupId>
             <artifactId>Slimefun4</artifactId>
-            <version>2c4f886</version>
+            <version>5dbe617205</version>
```
Comment (J3fftw1) [inline pom.xml:128]:
```suggestion
            <version>d191599</version>
```

**Review by J3fftw1 — APPROVED** [review]
LGTM

**Comment by Sefiraat** [comment]
Hold merging for now :)

**Comment by Sefiraat** [comment]
Ok, thought I found an issue but I was using the wrong build - all good ha

**Pull request merged**